metricbeat/module/consul/agent/_meta/fields.yml

```diff
@@ -0,0 +1,72 @@
+- name: agent
+  type: group
+  release: beta
+  description: >
+    Agent Metricset fetches metrics information from a Consul instance running as Agent
+  fields:
+    - name: raft
+      type: group
+      fields:
+        - name: commit_time
+          type: group
+          fields:
+            - name: ms
+              type: long
+              description: This tracks the average time in milliseconds it takes to commit a new entry to the transaction log of the leader
+        - name: apply
+          type: long
+          description: This metric gives the number of logs committed since the last interval.
+    - name: autopilot
+      type: group
+      fields:
+        - name: healthy
+          type: long
+          description: Tracks the overall health of the local server cluster. If all servers are considered healthy by Autopilot, this will be set to 1. If any are unhealthy, this will be 0.
+    - name: runtime
+      type: group
+      description: Runtime related metrics
+      fields:
+        - name: sys
+          type: group
+          fields:
+            - name: bytes
+              type: long
+              description: Total number of bytes of memory obtained from the OS.
+        - name: malloc_count
+          type: long
+          description: Heap objects allocated
+        - name: heap_objects
+          type: long
+          description: This measures the number of objects allocated on the heap and is a general memory pressure indicator. This may burst from time to time but should return to a steady state value.
```

Comment:
```suggestion
          description: Number of objects allocated on the heap. It is a general memory pressure indicator. This may burst from time to time but should return to a steady state value.
```

Reply:
Done, Thanks! :+1: